Common/Data/Custom/SmartInsider/SmartInsiderIntention.cs

```diff
@@ -276,64 +277,48 @@ public override BaseData Clone()
-                otherIntention.USDMarketCap == USDMarketCap &&
-                otherIntention.CompanyID == CompanyID &&
-                otherIntention.ICBIndustry == ICBIndustry &&
-                otherIntention.ICBSuperSector == ICBSuperSector &&
-                otherIntention.ICBSector == ICBSector &&
-                otherIntention.ICBSubSector == ICBSubSector &&
-                otherIntention.ICBCode == ICBCode &&
-                otherIntention.CompanyName == CompanyName &&
-                otherIntention.PreviousResultsAnnouncementDate == PreviousResultsAnnouncementDate &&
-                otherIntention.NextResultsAnnouncementsDate == NextResultsAnnouncementsDate &&
-                otherIntention.NextCloseBegin == NextCloseBegin &&
-                otherIntention.LastCloseEnded == LastCloseEnded &&
-                otherIntention.SecurityDescription == SecurityDescription &&
-                otherIntention.TickerCountry == TickerCountry &&
-                otherIntention.TickerSymbol == TickerSymbol &&
-                otherIntention.AnnouncementDate == AnnouncementDate &&
-                otherIntention.TimeReleased == TimeReleased &&
-                otherIntention.TimeProcessed == TimeProcessed &&
-                otherIntention.TimeReleasedUtc == TimeReleasedUtc &&
-                otherIntention.TimeProcessedUtc == TimeProcessedUtc &&
-                otherIntention.AnnouncedIn == AnnouncedIn &&
-
-                otherIntention.IntentionVia == IntentionVia &&
-                otherIntention.IntentionBy == IntentionBy &&
-                otherIntention.BuybackIntentionHoldingType == BuybackIntentionHoldingType &&
-                otherIntention.IntentionAmount == IntentionAmount &&
-                otherIntention.ValueCurrency == ValueCurrency &&
-                otherIntention.IntentionValue == IntentionValue &&
-                otherIntention.IntentionPercentage == IntentionPercentage &&
-                otherIntention.IntentionAuthorisationStartDate == IntentionAuthorisationStartDate &&
-                otherIntention.IntentionAuthorisationEndDate == IntentionAuthorisationEndDate &&
-                otherIntention.PriceCurrency == PriceCurrency &&
-                otherIntention.MinimumPrice == MinimumPrice &&
-                otherIntention.MaximumPrice == MaximumPrice &&
-                otherIntention.BuybackIntentionNoteText == BuybackIntentionNoteText;
+            return string.Join("\t",
+                TransactionID,
+                BuybackType,
+                LastUpdate.ToString("yyyyMMdd"),
+                LastIDsUpdate?.ToString("yyyyMMdd"),
+                ISIN,
+                USDMarketCap,
+                CompanyID,
+                ICBIndustry,
+                ICBSuperSector,
+                ICBSector,
+                ICBSubSector,
+                ICBCode,
+                CompanyName,
+                PreviousResultsAnnouncementDate?.ToString("yyyyMMdd"),
```

Comment:
Recall we have the `DateFormat` class, specifically designed for these cases.
https://github.com/QuantConnect/Lean/blob/90ab859d4714a48beaf4ac11bac26fa43dd3fbce/Common/Global.cs#L28